.github/workflows/release.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           # Replace in a specific file — adjust pattern and file path
           sed -i "s/2\.x\.x/${{ env.RELEASE_TAG }}/g" src/firecrest/config.py
           sed -i "s/2\.x\.x/${{ env.RELEASE_TAG }}/g" build/demo-launcher/Dockerfile
+          sed -i "s/2\.x\.x/${{ env.RELEASE_TAG }}/g" build/helm/firecrest-api/Chart.yaml
```

Comment:
Given in the Chart file there are multiple version values I would restrict the replace regex to appVersion only.
Something like: `sed -i "s/appVersion: 2\.x\.x/${{ env.RELEASE_TAG }}/g"`